kythe/cxx/verifier/assertion_ast.h

```diff
@@ -78,7 +104,9 @@ class SymbolTable {
   /// Maps `Symbol`s back to their original text.
   std::vector<const std::string*> reverse_map_;
   /// The text to use for unique() symbols.
-  std::string unique_symbol_ = "(unique)";
+  const std::string unique_symbol_;
+  /// Used for quoting strings - see assertions.lex:
+  const std::regex id_regex_;
```

Comment:
Making these const disables efficient move semantics and all assignment; please remove the `const` qualifier.